src/Sylius/Bundle/ApiBundle/Command/RegisterShopUser.php

```diff
@@ -47,4 +50,14 @@ public function __construct(
         $this->password = $password;
         $this->phoneNumber = $phoneNumber;
     }
+
+    public function getChannelCode(): string
+    {
+        return $this->channelCode;
+    }
+
+    public function setChannelCode(?string $channelCode): void
+    {
+        $this->channelCode = $channelCode;
```

Comment:
`@psalm-immutable` should go from the top of the class to the fields that were here previously (all of them except channelCode)